tests/models/test_serialized_dag.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Unit tests for SerializedDagModel."""
+
+import unittest
+
+from airflow import example_dags as example_dags_module
+from airflow.dag.serialization.serialized_dag import SerializedDAG
+from airflow.models import DagBag
+from airflow.models import SerializedDagModel as SDM
+from airflow.utils import db
+
+
+EXAMPLE_DAGS = [
```

Comment:
We should not hardcode this as well. 

That will allow us to test serialisation for new example dags as well. WDYT?